.github/workflows/build-native.yml

```diff
@@ -51,6 +51,10 @@ jobs:
         if: startsWith(matrix.build, 'android')
         run: echo "ANDROID_NDK=$ANDROID_NDK_LATEST_HOME" >> $GITHUB_ENV
 
+      - name: Downgrade XCode for MacOS
```

Comment:
We need to move this job to macos-12 for this to work.

Reply:
Using XCode 14.3.1 instead.